<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Publications featuring the pedsuite • pedsuite</title>
<script src="../../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../../deps/font-awesome-6.4.2/css/all.min.css" rel="stylesheet">
<link href="../../deps/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet">
<script src="../../deps/headroom-0.11.0/headroom.min.js"></script><script src="../../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../../deps/search-1.0.0/fuse.min.js"></script><script src="../../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../../pkgdown.js"></script><meta property="og:title" content="Publications featuring the pedsuite">
<meta name="google-site-verification" content="0EyxedeIXm4ThmZacSrfJ7L0z17Ulx0UKdK2Yo7p3wk">
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-light" data-bs-theme="light" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../../index.html">pedsuite</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.3.1</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item"><a class="nav-link" href="../../articles/pedsuite.html">Get started</a></li>
<li class="active nav-item"><a class="nav-link" href="../../articles/web_only/publications.html">Publications</a></li>
<li class="nav-item dropdown">
  <button class="nav-link dropdown-toggle" type="button" id="dropdown-shiny-apps" data-bs-toggle="dropdown" aria-expanded="false" aria-haspopup="true">Shiny apps</button>
  <ul class="dropdown-menu" aria-labelledby="dropdown-shiny-apps">
<li><a class="dropdown-item" href="../../articles/web_only/quickped.html">QuickPed</a></li>
    <li><a class="dropdown-item" href="../../articles/web_only/klink.html">KLINK</a></li>
    <li><a class="dropdown-item" href="../../articles/web_only/linkagelab.html">LinkageLab</a></li>
    <li><a class="dropdown-item" href="../../articles/web_only/ibdsim2shiny.html">ibdsim2-shiny</a></li>
    <li><a class="dropdown-item" href="../../articles/web_only/ibdClassifier.html">ibdClassifier</a></li>
  </ul>
</li>
<li class="nav-item"><a class="nav-link" href="../../articles/web_only/courses.html">Courses</a></li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/magnusdv/pedsuite/" aria-label="GitHub"><span class="fa fab fa-github fa-lg"></span></a></li>
<li class="nav-item"><a class="external-link nav-link" href="https://twitter.com/mdvigeland" aria-label="twitter account"><span class="fa fab fa-twitter fa-lg"></span></a></li>
      </ul>
</div>


  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">

      <h1>Publications featuring the pedsuite</h1>
            
      

      <div class="d-none name"><code>publications.Rmd</code></div>
    </div>

    
    
<div class="section level2">
<h2 id="book">Book<a class="anchor" aria-label="anchor" href="#book"></a>
</h2>
<p>The book <a href="https://www.elsevier.com/books/pedigree-analysis-in-r/vigeland/978-0-12-824430-2" class="external-link">Pedigree
Analysis in R</a> (Academic Press, 2021) is based on the
<strong>pedsuite</strong> and contains detailed explanations and
examples of how to use the different packages.</p>
<p><img src="pedinr-cover.png" width="30%" style="display: block; margin: auto;"></p>
<hr>
</div>
<div class="section level2">
<h2 id="papers">Papers<a class="anchor" aria-label="anchor" href="#papers"></a>
</h2>
<p>The following is a list of publications featuring
<strong>pedsuite</strong> packages. If you know of others, please let me
know!</p>
<div class="section level4">
<h4 id="pedsuite-multiple-packages">
<strong><code>pedsuite</code> (multiple packages)</strong><a class="anchor" aria-label="anchor" href="#pedsuite-multiple-packages"></a>
</h4>
<ul>
<li>MD Vigeland. <a href="https://doi.org/10.1186/s12859-022-04759-y" class="external-link">QuickPed: an online
tool for drawing pedigrees and analysing relatedness</a>. <em>BMC
Bioinf</em>, <strong>2022</strong>.</li>
<li>M Kruijver, J-A Bright. <a href="https://doi.org/10.1016/j.fsigen.2022.102746" class="external-link">A tool for
simulating single source and mixed DNA profiles</a>. <em>Forensic Sci
Int: Genetics</em>, <strong>2022</strong>.</li>
<li>NE Altınışık, DD Kazancı, et al. <a href="https://doi.org/10.1126/sciadv.abo3609" class="external-link">A genomic snapshot of
demographic and cultural dynamism in Upper Mesopotamia during the
Neolithic Transition</a>. <em>Science Advances</em>,
<strong>2022</strong>.</li>
</ul>
</div>
<div class="section level4">
<h4 id="dvir">
<strong><code>dvir</code></strong><a class="anchor" aria-label="anchor" href="#dvir"></a>
</h4>
<ul>
<li>MD Vigeland, T Egeland. <a href="https://doi.org/10.1038/s41598-021-93071-5" class="external-link">Joint DNA-based
disaster victim identification</a>. <em>Sci Rep</em>,
<strong>2021</strong>.</li>
</ul>
</div>
<div class="section level4">
<h4 id="forrel">
<strong><code>forrel</code></strong><a class="anchor" aria-label="anchor" href="#forrel"></a>
</h4>
<ul>
<li>FL Marsico, I Caridi. <a href="https://doi.org/10.1016/j.fsigen.2023.102891" class="external-link">Incorporating
non-genetic evidence in large scale missing person searches: A general
approach beyond filtering</a>. <em>Forensic Sci Int: Genetics</em>,
<strong>2023</strong>.</li>
<li>M Iungman, S Biagini, M Canteros et al. <a href="https://doi.org/10.1016/j.fsigss.2022.10.059" class="external-link">Empirical validation
of a family-member prioritization approach to maximize statistical power
in missing person cases</a>. <em>Forensic Sci Int: Genetics Suppl
Series</em>, <strong>2022</strong>.</li>
<li>A Chernomoretz, F Marsico, et al. <a href="https://doi.org/10.1016/j.fsigss.2022.10.008" class="external-link">Bayesian networks
for DNA-based kinship analysis: Functionality and validation of the
GENis missing person identification module</a>. Forensic Sci Int:
Genetics Suppl Series, <strong>2022</strong>.</li>
<li>HK Brustad, M Colucci, MA Jobling NA Sheehand, T Egeland. <a href="https://doi.org/10.1016/j.fsigen.2021.102562" class="external-link">Strategies for
pairwise searches in forensic kinship analysis</a>. <em>Forensic Sci
Int: Genetics</em>, <strong>2021</strong>.</li>
<li>FL Marsico, MD Vigeland, T Egeland, M Herrera Piñero. <a href="https://doi.org/10.1016/j.fsigen.2021.102519" class="external-link">Making decisions in
missing person identification cases with low statistical power</a>.
<em>Forensic Sci Int: Genetics</em>, <strong>2021</strong>.</li>
<li>HK Brustad, MD Vigeland, T Egeland. <a href="https://doi.org/10.1007/s00414-020-02426-6" class="external-link">Pairwise relatedness
testing in the context of inbreeding: expectation and variance of the
likelihood ratio</a>. <em>Int J Legal Med</em>,
<strong>2020</strong>.</li>
<li>MD Vigeland, FL Marsico, M Herrera Piñero, T Egeland. <a href="https://doi.org/10.1016/j.fsigen.2020.102376" class="external-link">Prioritising family
members for genotyping in missing person cases: A general approach
combining the statistical power of exclusion and inclusion</a>.
<em>Forensic Sci Int: Genetics</em>, <strong>2020</strong>.</li>
<li>HK Brustad, T Egeland. <a href="https://doi.org/10.1016/j.fsigss.2019.10.052" class="external-link">The impact of
ignoring inbreeding in pairwise kinship evaluations</a>. <em>Forensic
Sci Int: Genetics Suppl Series</em>, <strong>2019</strong>.</li>
<li>MD Vigeland, T Egeland. <a href="https://doi.org/10.1016/j.fsigss.2019.10.175" class="external-link">Handling founder
inbreeding in forensic kinship analysis</a>. <em>Forensic Sci Int:
Genetics Suppl Series</em>, <strong>2019</strong>.</li>
</ul>
</div>
<div class="section level4">
<h4 id="ibdsim-ibdsim2">
<strong><code>IBDsim</code></strong> /
<strong><code>ibdsim2</code></strong><a class="anchor" aria-label="anchor" href="#ibdsim-ibdsim2"></a>
</h4>
<ul>
<li>C Cavagnino, G Runfeldt, et al. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11074960/" class="external-link">Unearthing
who and Y at Harewood Cemetery and inference of George Washington’s
Y-chromosomal haplotype</a>. <em>iScience</em>,
<strong>2024</strong>.</li>
<li>H Vinje, HK Brustad, et al. <a href="https://doi.org/10.3389/fgene.2023.1289130" class="external-link">Classification of
breed combinations for slaughter pigs based on genotypes—modeling DNA
samples of crossbreeds as fuzzy sets from purebred founders</a>.
<em>Front Genet</em>, <strong>2023</strong>.</li>
<li>VM Narasimhan, R Rahbari, A Scally, et al. <a href="https://doi.org/10.1038/s41467-017-00323-y" class="external-link">Estimating the human
mutation rate from autozygous segments reveals population differences in
human mutational processes</a>. <em>Nat Commun</em>,
<strong>2017</strong>.</li>
<li>S Steinberg, S Gudmundsdottir, G Sveinbjornsson, et al. <a href="https://doi.org/10.1038/ng.3894" class="external-link">Truncating mutations in RBM12 are
associated with psychosis</a>. <em>Nat Genet</em>,
<strong>2017</strong>.</li>
</ul>
</div>
<div class="section level4">
<h4 id="paramlink-paramlink2">
<strong><code>paramlink</code></strong> /
<strong><code>paramlink2</code></strong><a class="anchor" aria-label="anchor" href="#paramlink-paramlink2"></a>
</h4>
<ul>
<li>Ø Bleka, L Prieto, P Gill. <a href="https://doi.org/10.1016/j.fsigen.2022.102771" class="external-link">EFMrep: An extension
of EuroForMix for improved combination of STR DNA mixture profiles</a>.
<em>Forensic Sci Int: Genetics</em>, <strong>2022</strong>.</li>
<li>Z Bai, N Zhang, JLiu, et al. <a href="https://doi.org/10.1016/j.fsigen.2022.102689" class="external-link">Identification of
missing persons through kinship analysis by microhaplotype sequencing of
single-source DNA and two-person DNA mixtures</a>. <em>Forensic Sci Int:
Genetics</em>, <strong>2022</strong>.</li>
<li>F Hendrickx, Z De Corte, G Sonet, et al. <a href="https://doi.org/10.1101/2021.02.09.430505" class="external-link">A masculinizing
supergene underlies an exaggerated male reproductive morph in a
spider</a>. <em>Nature Ecology &amp; Evolution</em>,
<strong>2022</strong>.</li>
<li>W Li, X Gu, C Liu, et al. <a href="https://doi.org/10.1038/s41467-021-21519-3" class="external-link">A synergetic effect of
BARD1 mutations on tumorigenesis</a>. <em>Nat Commun</em>,
<strong>2021</strong>.</li>
<li>LR Lopes, S Garcia-Hernández, M Lorenzini, et al. <a href="https://doi.org/10.1093/eurheartj/ehab424" class="external-link">Alpha-protein kinase 3
(ALPK3) truncating variants are a cause of autosomal dominant
hypertrophic cardiomyopathy</a>. <em>Eur Heart J</em>,
<strong>2021</strong>.</li>
<li>AL Junior, AG González, AR Vilela, et al. <a href="https://doi.org/10.1016/j.rec.2021.01.001" class="external-link">Genotype-phenotype
correlations in hypertrophic cardiomyopathy: a multicenter study in
Portugal and Spain of the TPM1 p.Arg21Leu variant</a>. <em>Rev Esp
Cardiol (English Ed)</em>, <strong>2021</strong>.</li>
<li>G Wu, J Ruan, J Liu, et al. <a href="https://doi.org/10.1161/JAHA.120.018236" class="external-link">Variant spectrum of
Formin Homology 2 Domain‐Containing 3 gene in Chinese patients with
hypertrophic cardiomyopathy</a>. <em>Journal of the American Heart
Association</em>, <strong>2021</strong>.</li>
<li>SL Rydning, A Dudesek, F Rimmele, et al. <a href="https://doi.org/10.1111/ene.13625" class="external-link">A novel heterozygous variant in
ERLIN2 causes autosomal dominant pure hereditary spastic paraplegia</a>.
<em>European journal of neurology</em>, <strong>2018</strong>.</li>
<li>P May, S Pichler, D Hartl, et al. <a href="https://doi.org/10.1212/NXG.0000000000000224" class="external-link">Rare ABCA7 variants
in 2 German families with Alzheimer disease</a>. <em>Neurology
Genetics</em>, <strong>2018</strong>.</li>
<li>JP Ochoa, M Sabater-Molina, JM García-Pinilla, et al. <a href="https://doi.org/10.1016/j.jacc.2018.10.001" class="external-link">Formin homology 2
domain containing 3 (FHOD3) is a genetic basis for hypertrophic
cardiomyopathy</a>. <em>Journal of the American College of
Cardiology</em>, <strong>2018</strong>.</li>
<li>AO Tillmar, D Kling, JM Butler, et al. <a href="https://doi.org/10.1016/j.fsigen.2017.05.005" class="external-link">DNA Commission of
the International Society for Forensic Genetics (ISFG): Guidelines on
the use of X-STRs in kinship analysis</a>. <em>Forensic Sci Int:
Genetics</em>, <strong>2017</strong>.</li>
<li>IL Mero, HH Mørk, Y Sheng, et al. <a href="https://doi.org/10.1093/hmg/ddx263" class="external-link">Homozygous <em>KIDINS220</em>
loss-of-function variants in fetuses with cerebral ventriculomegaly and
limb contractures</a>. <em>Human Molecular Genetics</em>,
<strong>2017</strong>.</li>
<li>D Kling, T Egeland, M Herrera Piñero, MD Vigeland. <a href="https://doi.org/10.1016/j.fsigen.2017.08.006" class="external-link">Evaluating the
statistical power of DNA-based identification, exemplified by ‘The
missing grandchildren of Argentina’</a>. <em>Forensic Sci Int:
Genetics</em>, <strong>2017</strong>.</li>
<li>G Dørum, D Kling, A Tillmar, MD Vigeland. <a href="https://doi.org/10.1007/s00414-015-1288-x" class="external-link">Mixtures with relatives
and linked markers</a>. <em>Int J Legal Med</em>,
<strong>2016</strong>.</li>
<li>M Szopa, AH Ludwig-Galezowska, et al. <a href="https://doi.org/10.1016/j.ejmg.2016.01.002" class="external-link">A family with the
Arg103Pro mutation in the NEUROD1 gene detected by next-generation
sequencing–clinical characteristics of mutation carriers</a>. <em>Eur J
Med Gen</em>, <strong>2016</strong>.</li>
<li>T Egeland, N Pinto, MD Vigeland. <a href="https://doi.org/10.1016/j.fsigen.2013.05.001" class="external-link">A general approach
to power calculation for relationship testing</a>. <em>Forensic Sci Int:
Genetics</em>, <strong>2014</strong>.</li>
<li>T Egeland, G Dørum, MD Vigeland, NA Sheehan. <a href="https://doi.org/10.1016/j.fsigen.2014.01.007" class="external-link">Mixtures with
relatives: A pedigree perspective</a>. <em>Forensic Sci Int:
Genetics</em>, <strong>2014</strong>.</li>
<li>G Park, J Gim, AR Kim, et al. <a href="https://doi.org/10.1186/1471-2164-14-191" class="external-link">Multiphasic analysis of
whole exome sequencing data identifies a novel mutation of ACTG1 in a
nonsyndromic hearing loss family</a>. <em>BMC Genomics</em>,
<strong>2013</strong>.</li>
</ul>
</div>
<div class="section level4">
<h4 id="pedbuildr">
<strong><code>pedbuildr</code></strong><a class="anchor" aria-label="anchor" href="#pedbuildr"></a>
</h4>
<ul>
<li>Y Wei, Q Zhu, H Wang, et al. <a href="https://doi.org/10.1016/j.fsigen.2024.103090" class="external-link">Pairwise kinship
inference and pedigree reconstruction using 91 microhaplotypes</a>.
<em>Forensic Sci Int: Genetics</em>, <strong>2022</strong>.</li>
</ul>
</div>
<div class="section level4">
<h4 id="pedprobr">
<strong><code>pedprobr</code></strong><a class="anchor" aria-label="anchor" href="#pedprobr"></a>
</h4>
<ul>
<li>S Bae, S Won, H Kim. <a href="https://doi.org/10.1007/s00414-020-02495-7" class="external-link">Selection and
evaluation of bi-allelic autosomal SNP markers for paternity testing in
Koreans</a>. <em>Int J Legal Med</em>, <strong>2021</strong>.</li>
</ul>
</div>
<div class="section level4">
<h4 id="pedtools">
<strong><code>pedtools</code></strong><a class="anchor" aria-label="anchor" href="#pedtools"></a>
</h4>
<ul>
<li>V Papin, G Gorjanc. <a href="https://doi.org/10.1101/2024.01.25.577259" class="external-link">Unlocking genome-based
prediction and selection in conifers: the key role of within-family
prediction accuracy</a>. Preprint, <strong>2024</strong>.</li>
</ul>
</div>
<div class="section level4">
<h4 id="quickped">
<strong><code>QuickPed</code></strong><a class="anchor" aria-label="anchor" href="#quickped"></a>
</h4>
<ul>
<li>I Koch, M Slovik, et al. <a href="https://doi.org/10.26508/lsa.202302258" class="external-link">USP27X variants underlying
X-linked intellectual disability disrupt protein function via distinct
mechanisms</a>. <em>Life Science Alliance</em>,
<strong>2024</strong>.</li>
<li>R Simon, K Elísabetardóttir, and G Lühke. <a href="">Analysis of
genetic variants for different horn phenotypes and their inheritance in
Icelandic sheep</a>. <em>Arch Anim Breed</em>,
<strong>2024</strong>.</li>
<li>Z Kolkova, P Durdik, et al. <a href="https://doi.org/10.3389/fped.2024.1339664" class="external-link">Identification of a
novel RPGR mutation associated with retinitis pigmentosa and primary
ciliary dyskinesia in a Slovak family: a case report</a>. <em>Front
Pediatr</em>, <strong>2024</strong>.</li>
<li>TA Vaz, RM Hassunuma, et al. <a href="https://doi.org/10.51189/integrar/rema/4390" class="external-link">QUICKPED: Um breve
guia de como criar rapidamente um heredograma utilizando um recurso
didático on-line gratuito</a>. <em>Rev. Multi. Educação e Meio Amb</em>,
<strong>2024</strong>.</li>
</ul>
</div>
<div class="section level4">
<h4 id="ribd">
<strong><code>ribd</code></strong><a class="anchor" aria-label="anchor" href="#ribd"></a>
</h4>
<ul>
<li>T Nomura. <a href="https://doi.org/10.3390/ani14131844" class="external-link">Ballou’s
Ancestral Inbreeding Coefficient: Formulation and New Estimate with
Higher Reliability</a>. <em>Animals</em>, <strong>2024</strong>.</li>
<li>MD Vigeland. <a href="https://doi.org/10.1093/g3journal/jkac326" class="external-link">Two-locus identity
coefficients in pedigrees</a>. <em>G3 Genes|Genomes|Genetics</em>,
<strong>2023</strong>.</li>
<li>M Kruijver, JM Curran. <a href="https://doi.org/10.1016/j.fsigen.2022.102748" class="external-link">The number of
alleles in DNA mixtures with related contributors</a>. <em>Forensic Sci
Int: Genetics</em>, <strong>2022</strong>.</li>
<li>A Sharma, S Szymczak, M Rühlemann, et al. <a href="https://doi.org/10.1080/19490976.2021.2024415" class="external-link">Linkage analysis
identifies novel genetic modifiers of microbiome traits in families with
inflammatory bowel disease</a>. <em>Gut Microbes</em>,
<strong>2022</strong>.</li>
<li>PJ Green, J Montera. <a href="https://doi.org/10.1111/rssc.12498" class="external-link">Inference about complex
relationships using peak height data from DNA mixtures</a>. <em>J R Stat
Soc Ser C Appl Stat</em>, <strong>2021</strong>.</li>
<li>MD Vigeland. <a href="https://doi.org/10.1007/s00285-020-01505-x" class="external-link">Relatedness
coefficients in pedigrees with inbred founders</a>. <em>Journal of
Mathematical Biology</em>, <strong>2020</strong>.</li>
</ul>
</div>
<div class="section level4">
<h4 id="segregatr">
<strong><code>segregatr</code></strong><a class="anchor" aria-label="anchor" href="#segregatr"></a>
</h4>
<ul>
<li>H-K Liu, H-L Hao, et al. <a href="https://doi.org/10.1007/s12264-024-01231-0" class="external-link">A Cysteinyl-tRNA
Synthetase Mutation Causes Novel Autosomal-Dominant Inheritance of a
Parkinsonism/Spinocerebellar-Ataxia Complex</a>. <em>Neurosci Bull</em>,
<strong>2024</strong>.</li>
<li>SA Schubert, D Ruano, et al. <a href="https://doi.org/10.1038/s41467-024-52551-8" class="external-link">Germline variant
affecting p53β isoforms predisposes to familial cancer</a>. <em>Nat
Commun</em>, <strong>2024</strong>.</li>
<li>C Carrizosa, DE Undlien, MD Vigeland. <a href="https://doi.org/10.1093/bioinformatics/btae201" class="external-link">shinyseg: a web
application for flexible cosegregation and sensitivity analysis</a>.
<em>Bioinformatics</em>, <strong>2024</strong>.</li>
<li>A Ratajska, MD Vigeland, et al. <a href="https://doi.org/10.1002/mgg3.2107" class="external-link">The use of segregation analysis
in interpretation of sequence variants in SMAD3: A case report</a>.
<em>Molec. Genetics &amp; Genomic Med</em>, <strong>2023</strong>.</li>
<li>PM Bjørnstad, R Aaløkken, et al. <a href="https://doi.org/10.1038/s41431-023-01494-7" class="external-link">A 39 kb structural
variant causing Lynch Syndrome detected by optical genome mapping and
nanopore sequencing</a>. <em>Eur J Med Gen</em>,
<strong>2023</strong>.</li>
</ul>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Created by Magnus Dehli Vigeland.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.1.</p>
</div>

    </footer>
</div>





  </body>
</html>
