<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Easy installation of the ped suite packages for pedigree analysis • pedsuite</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="bootstrap-toc.css">
<script src="bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="pkgdown.css" rel="stylesheet">
<script src="pkgdown.js"></script><meta property="og:title" content="Easy installation of the ped suite packages for pedigree analysis">
<meta property="og:description" content="The ped suite is a collection of packages for pedigree
    analysis, covering applications in forensic genetics, medical genetics
    and others. A detailed presentation of the ped suite is given in the
    book Pedigree Analysis in R (Vigeland, 2021, ISBN: 9780128244302).">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-home">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="index.html">pedsuite</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.0.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/magnusdv/pedsuite/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/mdvigeland">
    <span class="fab fa-twitter fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="contents col-md-9">

<div id="ped-suite" class="section level1">
<div class="page-header"><h1 class="hasAnchor">
<a href="#ped-suite" class="anchor"></a>ped suite</h1></div>
<!-- badges: start -->
<!-- badges: end -->
<p>The <strong>ped suite</strong> is a collection of R packages for pedigree analysis, covering a variety of applications across several fields. The packages are designed to work harmoniously together, often allowing complex tasks to be solved in a few lines of code.</p>
<p>The following picture gives an overview of how the <strong>ped suite</strong> is organised. At the center we find the <strong>pedtools</strong> package, on which all the others depend.</p>
<p><img src="reference/figures/logos-pedsuite-wheel_1.png" width="100%" style="display: block; margin: auto;"></p>
<div id="core-packages" class="section level4">
<h4 class="hasAnchor">
<a href="#core-packages" class="anchor"></a>Core packages</h4>
<p>The following packages are the most common, and frequently used by other <strong>ped suite</strong> packages. They are all automatically attached by the command <code><a href="https://magnusdv.github.io/pedsuite">library(pedsuite)</a></code>.</p>
<ul>
<li>
<a href="https://github.com/magnusdv/pedtools">pedtools</a>: Creating and working with pedigrees and marker data</li>
<li>
<a href="https://github.com/magnusdv/ribd">ribd</a>: Computation of pedigree-based relatedness coefficients</li>
<li>
<a href="https://github.com/magnusdv/pedprobr">pedprobr</a>: Marker probabilities and pedigree likelihoods</li>
<li>
<a href="https://github.com/magnusdv/pedmut">pedmut</a>: Mutation models for pedigree likelihood computations</li>
<li>
<a href="https://github.com/magnusdv/forrel">forrel</a>: Forensic pedigree analysis and relatedness analysis</li>
</ul>
</div>
<div id="specialised-packages" class="section level4">
<h4 class="hasAnchor">
<a href="#specialised-packages" class="anchor"></a>Specialised packages</h4>
<p>The following packages are devoted to special applications:</p>
<ul>
<li>
<a href="https://github.com/magnusdv/ibdsim2">ibdsim2</a>: Simulation of identity-by-descent sharing by family members</li>
<li>
<a href="https://github.com/thoree/dvir">dvir</a>: Disaster victim identification</li>
<li>
<a href="https://github.com/magnusdv/pedbuildr">pedbuildr</a>: Pedigree reconstruction</li>
<li>
<a href="https://github.com/magnusdv/paramlink2">paramlink2</a>: Parametric linkage analysis</li>
<li>
<a href="https://github.com/magnusdv/segregatr">segregatr</a>: Segregation analysis for clinical variant interpretation</li>
</ul>
</div>
</div>
<div id="book" class="section level1">
<h1 class="hasAnchor">
<a href="#book" class="anchor"></a>Book</h1>
<p>The book <em>Pedigree Analysis in R</em> (Academic Press, 2021) is based on the <strong>ped suite</strong> and contains detailed explanations and examples of how to use the different packages.</p>
<p><img src="reference/figures/pedinr-cover.jpg" width="30%" style="display: block; margin: auto;"></p>
<p><br> Check it out at <a href="https://www.elsevier.com/books/pedigree-analysis-in-r/vigeland/978-0-12-824430-2" class="uri">https://www.elsevier.com/books/pedigree-analysis-in-r/vigeland/978-0-12-824430-2</a>.</p>
</div>
<div id="papers" class="section level1">
<h1 class="hasAnchor">
<a href="#papers" class="anchor"></a>Papers</h1>
<p>Papers featuring <strong>ped suite</strong> packages:</p>
<div id="dvir" class="section level4">
<h4 class="hasAnchor">
<a href="#dvir" class="anchor"></a>dvir</h4>
<ul>
<li>MD Vigeland, T Egeland. Joint DNA-based disaster victim identification. <strong>2021</strong>. <a href="https://doi.org/10.21203/rs.3.rs-296414/v1" class="uri">https://doi.org/10.21203/rs.3.rs-296414/v1</a>. (<em>In review.</em>)</li>
</ul>
</div>
<div id="forrel" class="section level4">
<h4 class="hasAnchor">
<a href="#forrel" class="anchor"></a>forrel</h4>
<ul>
<li>FL Marsico, MD Vigeland, T Egeland, M Herrera Piñero. Making decisions in missing person identification cases with low statistical power. <em>Forensic Science International: Genetics</em>, <strong>2021</strong>. <a href="https://doi.org/10.1016/j.fsigen.2021.102519" class="uri">https://doi.org/10.1016/j.fsigen.2021.102519</a>.</li>
<li>HK Brustad, MD Vigeland, T Egeland. Pairwise relatedness testing in the context of inbreeding: expectation and variance of the likelihood ratio. <em>International Journal of Legal Medicine</em>, <strong>2020</strong>. <a href="https://doi.org/10.1007/s00414-020-02426-6" class="uri">https://doi.org/10.1007/s00414-020-02426-6</a>.</li>
<li>MD Vigeland, FL Marsico, M Herrera Piñero, T Egeland. Prioritising family members for genotyping in missing person cases: A general approach combining the statistical power of exclusion and inclusion. <em>Forensic Science International: Genetics</em>, <strong>2020</strong>. <a href="https://doi.org/10.1016/j.fsigen.2020.102376" class="uri">https://doi.org/10.1016/j.fsigen.2020.102376</a>.</li>
<li>HK Brustad, T Egeland. The impact of ignoring inbreeding in pairwise kinship evaluations. <em>Forensic Science International: Genetics Supplement Series</em>, <strong>2019</strong>. <a href="https://doi.org/10.1016/j.fsigss.2019.10.052" class="uri">https://doi.org/10.1016/j.fsigss.2019.10.052</a>.</li>
<li>MD Vigeland, T Egeland. Handling founder inbreeding in forensic kinship analysis. <em>Forensic Science International: Genetics Supplement Series</em>, <strong>2019</strong>. <a href="https://doi.org/10.1016/j.fsigss.2019.10.175" class="uri">https://doi.org/10.1016/j.fsigss.2019.10.175</a>.</li>
</ul>
</div>
<div id="paramlink" class="section level4">
<h4 class="hasAnchor">
<a href="#paramlink" class="anchor"></a>paramlink</h4>
<ul>
<li>G Wu, J Ruan, J Liu, C Zhang, L Kang, J Wang, Y Zou, L Song. Variant spectrum of Formin Homology 2 Domain‐Containing 3 gene in Chinese patients with hypertrophic cardiomyopathy. <em>Journal of the American Heart Association</em>, <strong>2021</strong>. <a href="https://doi.org/10.1161/JAHA.120.018236" class="uri">https://doi.org/10.1161/JAHA.120.018236</a>.</li>
<li>F Hendrickx, Z De Corte, G Sonet, SM Van Belleghem, S Köstlbacher, C Vangestel. A masculinizing supergene underlies an exaggerated male reproductive morph in a spider. <em>bioRxiv</em>, <strong>2021</strong>. <a href="https://doi.org/10.1101/2021.02.09.430505" class="uri">https://doi.org/10.1101/2021.02.09.430505</a>.</li>
<li>SL Rydning, A Dudesek, F Rimmele, C Funke, S Krüger, S Biskup, MD Vigeland, HS Hjorthaug, Y Sejersted, C Tallaksen, KK Selmer, C Kamm. A novel heterozygous variant in ERLIN2 causes autosomal dominant pure hereditary spastic paraplegia. <em>European journal of neurology</em>, <strong>2018</strong>. <a href="https://doi.org/10.1111/ene.13625" class="uri">https://doi.org/10.1111/ene.13625</a>.</li>
<li>P May, S Pichler, D Hartl, DR Bobbili, M Mayhaus, C Spaniol, A Kurz, R Balling, JG Schneider, M Riemenschneider. Rare ABCA7 variants in 2 German families with Alzheimer disease. <em>Neurology Genetics</em>, <strong>2018</strong>. <a href="https://doi.org/10.1212/NXG.0000000000000224" class="uri">https://doi.org/10.1212/NXG.0000000000000224</a>.</li>
<li>JP Ochoa, M Sabater-Molina, JM García-Pinilla, J Mogensen, A Restrepo-Córdoba, J Palomino-Doza, E Villacorta, M Martinez-Moreno, J Ramos-Maqueda, E Zorio, ML Peña-Peña, PE García-Granja, JF Rodríguez-Palomares, IJ Cárdenas-Reyes, MM de la Torre-Carpente, A Bautista-Pavés, MM Akhtar, MN Cicerchia, R Bilbao-Quesada, MV Mogollón-Jimenez, J Salazar-Mendiguchía, JMM Latorre, B Arnaez, I Olavarri-Miguel, ME Fuentes-Cañamero, A Lamounier, JMP Ruiz, V Climent-Payá, I Pérez-Sanchez, JP Trujillo-Quintero, LR Lopes, A Repáraz-Andrade, R Marín-Iglesias, A Rodriguez-Vilela, M Sandín-Fuentes, JA Garrote, A Cortel-Fuster, M Lopez-Garrido, A Fontalba-Romero, T Ripoll-Vera, I Llano-Rivas, X Fernandez-Fernandez, M Isidoro-García, D Garcia-Giustiniani, R Barriales-Villa, M Ortiz-Genga, P García-Pavía, PM Elliott, JR Gimeno, L Monserrat. Formin Homology 2 Domain Containing 3 (FHOD3) Is a Genetic Basis for Hypertrophic Cardiomyopathy. <em>Journal of the American College of Cardiology</em>, <strong>2018</strong>. <a href="https://doi.org/10.1016/j.jacc.2018.10.001" class="uri">https://doi.org/10.1016/j.jacc.2018.10.001</a>.</li>
<li>AO Tillmar, D Kling, JM Butler, W Parson, M Prinz, PM Schneider, T Egeland, L Gusmão. DNA Commission of the International Society for Forensic Genetics (ISFG): Guidelines on the use of X-STRs in kinship analysis. <em>Forensic Science International: Genetics</em>, <strong>2017</strong>. <a href="https://doi.org/10.1016/j.fsigen.2017.05.005" class="uri">https://doi.org/10.1016/j.fsigen.2017.05.005</a>
</li>
<li>IL Mero, HH Mørk, Y Sheng, A Blomhoff, GL Opheim, Aa Erichsen, MD Vigeland, KK Selmer. Homozygous <em>KIDINS220</em> loss-of-function variants in fetuses with cerebral ventriculomegaly and limb contractures. <em>Human Molecular Genetics</em>, <strong>2017</strong>. <a href="https://doi.org/10.1093/hmg/ddx263" class="uri">https://doi.org/10.1093/hmg/ddx263</a>.</li>
<li>D Kling, T Egeland, M Herrera Piñero, MD Vigeland. Evaluating the statistical power of DNA-based identification, exemplified by ‘The missing grandchildren of Argentina’. <em>Forensic Science International: Genetics</em>, <strong>2017</strong>. <a href="https://doi.org/10.1016/j.fsigen.2017.08.006" class="uri">https://doi.org/10.1016/j.fsigen.2017.08.006</a>.</li>
<li>G Dørum, D Kling, A Tillmar, MD Vigeland. Mixtures with relatives and linked markers. <em>International Journal of Legal Medicine</em>, <strong>2016</strong>. 10.1007/s00414-015-1288-x.</li>
<li>T Egeland, N Pinto, MD Vigeland. A general approach to power calculation for relationship testing. <em>Forensic Science International: Genetics</em>, <strong>2014</strong>. <a href="https://doi.org/10.1016/j.fsigen.2013.05.001" class="uri">https://doi.org/10.1016/j.fsigen.2013.05.001</a>.</li>
<li>T Egeland, G Dørum, MD Vigeland, NA Sheehan. Mixtures with relatives: A pedigree perspective. <em>Forensic Science International: Genetics</em>, <strong>2014</strong>. <a href="https://doi.org/10.1016/j.fsigen.2014.01.007" class="uri">https://doi.org/10.1016/j.fsigen.2014.01.007</a>.</li>
<li>G Park, J Gim, AR Kim, KH Han, HS Kim, SH Oh, T Park, WY Park, BY Choi. Multiphasic analysis of whole exome sequencing data identifies a novel mutation of ACTG1 in a nonsyndromic hearing loss family. <em>BMC Genomics</em>, <strong>2013</strong>. <a href="https://doi.org/10.1186/1471-2164-14-191" class="uri">https://doi.org/10.1186/1471-2164-14-191</a>.</li>
</ul>
</div>
<div id="ribd" class="section level4">
<h4 class="hasAnchor">
<a href="#ribd" class="anchor"></a>ribd</h4>
<ul>
<li>MD Vigeland. Relatedness coefficients in pedigrees with inbred founders. <em>Journal of Mathematical Biology</em>, <strong>2020</strong>. <a href="https://doi.org/10.1007/s00285-020-01505-x" class="uri">https://doi.org/10.1007/s00285-020-01505-x</a>.</li>
<li>PJ Green, J Montera. Inference about complex relationships using peak height data. <em>arXiv:2005.09365</em>, <strong>2020</strong>.</li>
</ul>
</div>
<div id="installation" class="section level2">
<h2 class="hasAnchor">
<a href="#installation" class="anchor"></a>Installation</h2>
<p>You can install <strong>pedsuite</strong> from <a href="https://github.com/magnusdv/pedsuite">GitHub</a> with:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># install.packages("devtools")</span>
<span class="fu">devtools</span><span class="fu">::</span><span class="fu">install_github</span><span class="op">(</span><span class="st">"magnusdv/pedsuite"</span><span class="op">)</span></code></pre></div>
</div>
</div>

  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <div class="links">
<h2>Links</h2>
<ul class="list-unstyled">
<li>Browse source code at <br><a href="https://github.com/magnusdv/pedsuite/">https://​github.com/​magnusdv/​pedsuite/​</a>
</li>
<li>Report a bug at <br><a href="https://github.com/magnusdv/pedsuite/issues">https://​github.com/​magnusdv/​pedsuite/​issues</a>
</li>
</ul>
</div>
<div class="license">
<h2>License</h2>
<ul class="list-unstyled">
<li><a href="LICENSE.html">Full license</a></li>
<li><small>GPL (&gt;= 3)</small></li>
</ul>
</div>
<div class="developers">
<h2>Developers</h2>
<ul class="list-unstyled">
<li>Magnus Dehli Vigeland <br><small class="roles"> Author, maintainer </small> <a href="https://orcid.org/0000-0002-9134-4962" target="orcid.widget" aria-label="ORCID"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
<li>Thore Egeland <br><small class="roles"> Author </small> <a href="https://orcid.org/0000-0002-3465-8885" target="orcid.widget" aria-label="ORCID"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
</ul>
</div>

  </div>
</div>


      <footer><div class="copyright">
  <p>Developed by Magnus Dehli Vigeland, Thore Egeland.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
