<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en-GB">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Getting started with the pedsuite â€¢ pedsuite</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/font-awesome-6.5.2/css/all.min.css" rel="stylesheet">
<link href="../deps/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet">
<script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Getting started with the pedsuite">
<meta name="google-site-verification" content="0EyxedeIXm4ThmZacSrfJ7L0z17Ulx0UKdK2Yo7p3wk">
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-light" data-bs-theme="light" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">pedsuite</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.4.0</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="active nav-item"><a class="nav-link" href="../articles/pedsuite.html">Get started</a></li>
<li class="nav-item"><a class="nav-link" href="../articles/web_only/publications.html">Publications</a></li>
<li class="nav-item dropdown">
  <button class="nav-link dropdown-toggle" type="button" id="dropdown-shiny-apps" data-bs-toggle="dropdown" aria-expanded="false" aria-haspopup="true">Shiny apps</button>
  <ul class="dropdown-menu" aria-labelledby="dropdown-shiny-apps">
<li><a class="dropdown-item" href="../articles/web_only/quickped.html">QuickPed</a></li>
    <li><a class="dropdown-item" href="../articles/web_only/klink.html">KLINK</a></li>
    <li><a class="dropdown-item" href="../articles/web_only/linkagelab.html">LinkageLab</a></li>
    <li><a class="dropdown-item" href="../articles/web_only/ibdsim2shiny.html">ibdsim2-shiny</a></li>
    <li><a class="dropdown-item" href="../articles/web_only/ibdClassifier.html">ibdClassifier</a></li>
  </ul>
</li>
<li class="nav-item"><a class="nav-link" href="../articles/web_only/events.html">Events</a></li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/magnusdv/pedsuite/" aria-label="GitHub"><span class="fa fab fa-github fa-lg"></span></a></li>
      </ul>
</div>


  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">

      <h1>Getting started with the pedsuite</h1>
            
      

      <div class="d-none name"><code>pedsuite.Rmd</code></div>
    </div>

    
    
<div class="section level2">
<h2 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h2>
<p>The following command installs the latest official versions of the
<strong>pedsuite</strong> packages:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"pedsuite"</span><span class="op">)</span></span></code></pre></div>
<p>Alternatively, you can install the development versions from
GitHub:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># install.packages("devtools")</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu">install_github</span><span class="op">(</span><span class="st">"magnusdv/pedsuite"</span><span class="op">)</span></span></code></pre></div>
<p>If you only need a few of the packages, you may choose to install
them individually instead of the entire collection.</p>
<hr>
</div>
<div class="section level2">
<h2 id="a-quick-tour">A quick tour<a class="anchor" aria-label="anchor" href="#a-quick-tour"></a>
</h2>
<p>The aim of this vignette is to illustrate a few of the possibilities
of the <strong>pedsuite</strong> packages. It should be noted that we
are barely scratching the surface here; in particular several packages
are not even mentioned. For a more comprehensive overview, I recommend
the <a href="https://magnusdv.github.io/pedsuite/articles/web_only/publications.html#book">book</a>.</p>
<p>To get started we load the <strong>pedsuite</strong> package, which
is a convenient shortcut for loading all the <a href="https://magnusdv.github.io/pedsuite/#core-packages">core
packages</a>, making their methods available in the current R
session.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://magnusdv.github.io/pedsuite/">pedsuite</a></span><span class="op">)</span></span>
<span><span class="co">#&gt; Loading required package: forrel</span></span>
<span><span class="co">#&gt; Loading required package: pedtools</span></span>
<span><span class="co">#&gt; Loading required package: pedprobr</span></span>
<span><span class="co">#&gt; Loading required package: ribd</span></span>
<span><span class="co">#&gt; Loading required package: verbalisr</span></span></code></pre></div>
<div class="section level3">
<h3 id="create-a-pedigree-pedtools">1. Create a pedigree (<strong>pedtools</strong>)<a class="anchor" aria-label="anchor" href="#create-a-pedigree-pedtools"></a>
</h3>
<p>We begin by creating and plotting a pedigree with a child of first
cousins:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">x</span> <span class="op">=</span> <span class="fu"><a href="https://magnusdv.github.io/pedsuite/reference/ped_basic.html">cousinPed</a></span><span class="op">(</span><span class="fl">1</span>, child <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">x</span><span class="op">)</span></span></code></pre></div>
<p><img src="pedsuite_files/figure-html/fc-ped1-1.png" class="r-plt" width="40%" style="display: block; margin: auto;"></p>
<p>For symmetry let us change the sex of individual 3. We also take the
opportunity to showcase some of the plot options (many more are
available - see the help page by typing <code><a href="https://magnusdv.github.io/pedsuite/reference/plotmethods.html">?plotmethods</a></code>):</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">x</span> <span class="op">=</span> <span class="fu"><a href="https://magnusdv.github.io/pedsuite/reference/getSex.html">swapSex</a></span><span class="op">(</span><span class="va">x</span>, ids <span class="op">=</span> <span class="fl">3</span><span class="op">)</span></span>
<span><span class="co">#&gt; Changing sex of spouses as well: 4</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">x</span>, </span>
<span>     hatched <span class="op">=</span> <span class="fl">9</span>, </span>
<span>     carrier <span class="op">=</span> <span class="fl">7</span><span class="op">:</span><span class="fl">8</span>,</span>
<span>     fill <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>pink <span class="op">=</span> <span class="fl">1</span><span class="op">)</span>, </span>
<span>     textAnnot <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>inside <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"1"</span> <span class="op">=</span> <span class="st">"?"</span><span class="op">)</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<p><img src="pedsuite_files/figure-html/fc-ped2-1.png" class="r-plt" width="40%" style="display: block; margin: auto;"></p>
</div>
<div class="section level3">
<h3 id="calculate-the-inbreeding-coefficient-ribd">2. Calculate the inbreeding coefficient (<strong>ribd</strong>)<a class="anchor" aria-label="anchor" href="#calculate-the-inbreeding-coefficient-ribd"></a>
</h3>
<p>The inbreeding coefficient
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>f</mi><annotation encoding="application/x-tex">f</annotation></semantics></math>
of a pedigree member is defined as the probability of
<em>autozygosity</em> at a random autosomal locus. That is, the
probability that the two homologous alleles have the same origin within
the pedigree.</p>
<p>For a child of first cousins one can work out by pen and paper that
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>f</mi><mo>=</mo><mn>1</mn><mi>/</mi><mn>16</mn></mrow><annotation encoding="application/x-tex">f = 1/16</annotation></semantics></math>.
Alternatively, we can calculate it with the <strong>ribd</strong>
function <code><a href="https://magnusdv.github.io/pedsuite/reference/inbreeding.html">inbreeding()</a></code>.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://magnusdv.github.io/pedsuite/reference/inbreeding.html">inbreeding</a></span><span class="op">(</span><span class="va">x</span>, ids <span class="op">=</span> <span class="fl">9</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 0.0625</span></span></code></pre></div>
<p>The output agrees with
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>f</mi><mo>=</mo><mn>1</mn><mi>/</mi><mn>16</mn></mrow><annotation encoding="application/x-tex">f = 1/16</annotation></semantics></math>.</p>
</div>
<div class="section level3">
<h3 id="realised-inbreeding-ibdsim2">3. Realised inbreeding (<strong>ibdsim2</strong>)<a class="anchor" aria-label="anchor" href="#realised-inbreeding-ibdsim2"></a>
</h3>
<p>For any particular child of first cousins, the actual autozygous
fraction of the genome (except X &amp; Y) is called the coefficient of
<em>realised inbreeding</em>, denoted
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>f</mi><mi>R</mi></msub><annotation encoding="application/x-tex">f_R</annotation></semantics></math>.
This may deviate substantially from the pedigree-based expectation
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>f</mi><mo>=</mo><mn>1</mn><mi>/</mi><mn>16</mn></mrow><annotation encoding="application/x-tex">f = 1/16</annotation></semantics></math>.</p>
<p>We can simulate the distribution of
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>f</mi><mi>R</mi></msub><annotation encoding="application/x-tex">f_R</annotation></semantics></math>
with the <strong>ibdsim2</strong> package. Since this is not a core
package we must load it separately.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/magnusdv/ibdsim2" class="external-link">ibdsim2</a></span><span class="op">)</span></span></code></pre></div>
<p>First, we use the function <code><a href="https://magnusdv.github.io/pedsuite/reference/ibdsim.html">ibdsim()</a></code> to simulate the
recombination process in the entire pedigree, 200 times:</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">sims</span> <span class="op">=</span> <span class="fu"><a href="https://magnusdv.github.io/pedsuite/reference/ibdsim.html">ibdsim</a></span><span class="op">(</span><span class="va">x</span>, N <span class="op">=</span> <span class="fl">200</span>, seed <span class="op">=</span> <span class="fl">123</span><span class="op">)</span></span>
<span><span class="co">#&gt; Simulation parameters:</span></span>
<span><span class="co">#&gt; Simulations  : 200</span></span>
<span><span class="co">#&gt; Chromosomes  : 1-22</span></span>
<span><span class="co">#&gt; Genome length: 2875 Mb</span></span>
<span><span class="co">#&gt;                2602.29 cM (male)</span></span>
<span><span class="co">#&gt;                4180.42 cM (female)</span></span>
<span><span class="co">#&gt; Recomb model : chi</span></span>
<span><span class="co">#&gt; Target indivs: 1-9</span></span>
<span><span class="co">#&gt; Skip recomb  : -</span></span>
<span><span class="co">#&gt; Total time used: 2.03 secs</span></span></code></pre></div>
<p>Now extract the autozygous segments of each simulation.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">fr</span> <span class="op">=</span> <span class="fu"><a href="https://magnusdv.github.io/pedsuite/reference/realised.html">realisedInbreeding</a></span><span class="op">(</span><span class="va">sims</span>, id <span class="op">=</span> <span class="fl">9</span><span class="op">)</span></span></code></pre></div>
<p>Here is a summary of the first 6 simulations, including the number of
segments and various length statistics:</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">fr</span><span class="op">$</span><span class="va">perSimulation</span><span class="op">)</span></span>
<span><span class="co">#&gt;   nSeg  meanLen   totLen   maxLen    minLen      fReal</span></span>
<span><span class="co">#&gt; 1   11 19.04942 209.5437 37.56569 0.8072431 0.06178760</span></span>
<span><span class="co">#&gt; 2   12 14.17475 170.0970 48.43677 1.3413298 0.05015605</span></span>
<span><span class="co">#&gt; 3   12 12.87744 154.5293 26.11588 0.1475564 0.04556565</span></span>
<span><span class="co">#&gt; 4   19 17.43950 331.3506 58.64604 1.2084099 0.09770450</span></span>
<span><span class="co">#&gt; 5   16 16.96611 271.4577 44.94334 1.0386058 0.08004404</span></span>
<span><span class="co">#&gt; 6   13 17.84722 232.0139 49.91368 0.9532618 0.06841334</span></span></code></pre></div>
<p>And here is a histogram of the realised inbreeding coefficients
(given in the right-most column above):</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/graphics/hist.html" class="external-link">hist</a></span><span class="op">(</span><span class="va">fr</span><span class="op">$</span><span class="va">perSimulation</span><span class="op">$</span><span class="va">fReal</span>, xlim <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">0</span>, <span class="fl">0.15</span><span class="op">)</span>, breaks <span class="op">=</span> <span class="fl">16</span>, xlab <span class="op">=</span> <span class="st">"f_R"</span>, main <span class="op">=</span> <span class="cn">NULL</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Expected value</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/graphics/abline.html" class="external-link">abline</a></span><span class="op">(</span>v <span class="op">=</span> <span class="fl">1</span><span class="op">/</span><span class="fl">16</span>, col <span class="op">=</span> <span class="fl">2</span><span class="op">)</span></span></code></pre></div>
<p><img src="pedsuite_files/figure-html/fr-hist-1.png" class="r-plt" width="80%" style="display: block; margin: auto;"></p>
<p>As we see, the distribution centres around the expectation
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>f</mi><mo>=</mo><mn>1</mn><mi>/</mi><mn>16</mn><mo>=</mo><mn>0.0625</mn></mrow><annotation encoding="application/x-tex">f = 1/16 = 0.0625</annotation></semantics></math>
(red vertical line) but has substantial spread. The sample standard
deviation can be found in <code>fr$stDev</code>, which in our case is
0.02.</p>
</div>
<div class="section level3">
<h3 id="marker-simulation-forrel">4. Marker simulation (<strong>forrel</strong>)<a class="anchor" aria-label="anchor" href="#marker-simulation-forrel"></a>
</h3>
<p>Note that everything we have done so far has been purely theoretical,
with no markers involved. In medical and forensic applications we
usually work with genetic data in the form of marker genotypes, so let
us simulate such a dataset for our family.</p>
<p>The <code><a href="https://rdrr.io/pkg/forrel/man/markerSim.html" class="external-link">markerSim()</a></code> function of the <strong>forrel</strong>
package simulates the genotypes of pedigree members for a specific type
of markers. For instance, here we produce 500 SNPs with alleles
<code>A</code> and <code>B</code> (equally frequent, by default):</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">y</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/forrel/man/markerSim.html" class="external-link">markerSim</a></span><span class="op">(</span><span class="va">x</span>, N <span class="op">=</span> <span class="fl">500</span>, alleles <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"A"</span>, <span class="st">"B"</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="co">#&gt; Unconditional simulation of 500 autosomal markers.</span></span>
<span><span class="co">#&gt; Individuals: 1, 2, 3, 4, 5, 6, 7, 8, 9</span></span>
<span><span class="co">#&gt; Allele frequencies:</span></span>
<span><span class="co">#&gt;    A   B</span></span>
<span><span class="co">#&gt;  0.5 0.5</span></span>
<span><span class="co">#&gt; Mutation model: No </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Simulation finished.</span></span>
<span><span class="co">#&gt; Calls to `likelihood()`: 0.</span></span>
<span><span class="co">#&gt; Total time used: 0 seconds.</span></span></code></pre></div>
<p>We can see the genotypes of the first few markers by printing
<code>y</code> to the console.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">y</span></span>
<span><span class="co">#&gt;  id fid mid sex &lt;1&gt; &lt;2&gt; &lt;3&gt; &lt;4&gt; &lt;5&gt;</span></span>
<span><span class="co">#&gt;   1   *   *   1 A/B B/B A/A B/B A/A</span></span>
<span><span class="co">#&gt;   2   *   *   2 A/B A/B A/B A/A A/B</span></span>
<span><span class="co">#&gt;   3   1   2   2 A/B A/B A/B A/B A/B</span></span>
<span><span class="co">#&gt;   4   *   *   1 A/B A/B B/B A/B A/B</span></span>
<span><span class="co">#&gt;   5   1   2   1 B/B A/B A/B A/B A/A</span></span>
<span><span class="co">#&gt;   6   *   *   2 A/B B/B A/A A/B A/A</span></span>
<span><span class="co">#&gt;   7   4   3   1 B/B A/A B/B A/B A/B</span></span>
<span><span class="co">#&gt;   8   5   6   2 A/B A/B A/B B/B A/A</span></span>
<span><span class="co">#&gt;   9   7   8   1 B/B A/A B/B B/B A/A</span></span>
<span><span class="co">#&gt; Only 5 (out of 500) markers are shown.</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="inference-of-pairwise-relationships-forrel">5. Inference of pairwise relationships
(<strong>forrel</strong>)<a class="anchor" aria-label="anchor" href="#inference-of-pairwise-relationships-forrel"></a>
</h3>
<p>If this was a real dataset, a natural quality control step would be
to check correctness of the pedigree. One way to do this is to use the
data to estimate each pairwise relationship, and compare the result with
the pedigree. The function <code><a href="https://rdrr.io/pkg/forrel/man/checkPairwise.html" class="external-link">checkPairwise()</a></code> does all of
this, and presents the result in a relationship triangle.</p>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/forrel/man/checkPairwise.html" class="external-link">checkPairwise</a></span><span class="op">(</span><span class="va">y</span><span class="op">)</span></span>
<span><span class="co">#&gt; Excluding inbred individuals: 9</span></span></code></pre></div>
<p><img src="pedsuite_files/figure-html/check-pairwise-1.png" class="r-plt" width="60%" style="display: block; margin: auto;"></p>
<p>The plot shows that all pairwise estimates are near their expected
location in the triangle.</p>
<hr>
</div>
</div>
<div class="section level2">
<h2 id="where-to-go-from-here">Where to go from here<a class="anchor" aria-label="anchor" href="#where-to-go-from-here"></a>
</h2>
<p>If you enjoyed this quick tour and would like more details, you
should check out the GitHub README files of the packages that interest
you, for instance:</p>
<ul>
<li>
<a href="https://github.com/magnusdv/pedtools" class="external-link">pedtools</a>:
Creating and working with pedigrees and marker data</li>
<li>
<a href="https://github.com/magnusdv/forrel" class="external-link">forrel</a>: Forensic
pedigree analysis and relatedness analysis</li>
<li>
<a href="https://github.com/magnusdv/ribd" class="external-link">ribd</a>: Computation of
pedigree-based relatedness coefficients</li>
<li>
<a href="https://github.com/magnusdv/verbalisr" class="external-link">verbalisr</a>:
Verbal descriptions of pedigree relationships</li>
<li>
<a href="https://github.com/magnusdv/ibdsim2" class="external-link">ibdsim2</a>:
Simulation of identity-by-descent sharing by family members</li>
<li>
<a href="https://github.com/magnusdv/dvir" class="external-link">dvir</a>: Disaster victim
identification</li>
</ul>
<p>Try the interactive pedigree builder <strong>QuickPed</strong>: <a href="https://magnusdv.shinyapps.io/quickped" class="external-link uri">https://magnusdv.shinyapps.io/quickped</a>.</p>
<p>The <a href="https://cran.r-project.org/package=pedtools/vignettes/pedtools.html" class="external-link">pedtools
vignette</a> contains details on pedigree objects in R and how to plot
and manipulate them.</p>
</div>
  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Created by Magnus Dehli Vigeland.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.2.0.</p>
</div>

    </footer>
</div>





  </body>
</html>
