<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Getting started with the ped suite • pedsuite</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Getting started with the ped suite">
<meta property="og:description" content="pedsuite">
<meta name="google-site-verification" content="0EyxedeIXm4ThmZacSrfJ7L0z17Ulx0UKdK2Yo7p3wk">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">pedsuite</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.0.4.9002</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/pedsuite.html">Get started</a>
</li>
<li>
  <a href="../articles/web_only/publications.html">Publications</a>
</li>
<li>
  <a href="../articles/web_only/quickped.html">QuickPed</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Courses
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/web_only/courses.html">Overview</a>
    </li>
    <li class="divider">
    <li>
      <a href="../articles/web_only/course-isfg2021.html">ISFG Summer School 2021</a>
    </li>
    <li>
      <a href="https://familias.name/norbisRelatedness/" class="external-link">NORBIS 2020</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/magnusdv/pedsuite/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/mdvigeland" class="external-link">
    <span class="fab fa-twitter fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="pedsuite_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Getting started with the ped suite</h1>
            
      
      
      <div class="hidden name"><code>pedsuite.Rmd</code></div>

    </div>

    
    
<div id="installation" class="section level2">
<h2 class="hasAnchor">
<a href="#installation" class="anchor" aria-hidden="true"></a>Installation</h2>
<p>The following command installs the latest official versions of the <strong>ped suite</strong> packages:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"pedsuite"</span><span class="op">)</span></code></pre></div>
<p>Alternatively, you can install the development versions from GitHub:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># install.packages("devtools")</span>
<span class="fu">devtools</span><span class="fu">::</span><span class="fu">install_github</span><span class="op">(</span><span class="st">"magnusdv/pedsuite"</span><span class="op">)</span></code></pre></div>
<p>If you only need a few of the packages, you may choose to install them individually instead of the entire collection.</p>
<hr>
</div>
<div id="a-quick-tour" class="section level2">
<h2 class="hasAnchor">
<a href="#a-quick-tour" class="anchor" aria-hidden="true"></a>A quick tour</h2>
<p>The aim of this vignette is to illustrate just a few of the possibilities of the <strong>ped suite</strong> packages. Of course, we barely scratch the surface here, and several packages are not even mentioned. For a more comprehensive overview, I recommend the <a href="https://magnusdv.github.io/pedsuite/articles/web_only/publications.html#book">book</a>.</p>
<p>To get started we load the <strong>pedsuite</strong> package, which is a convenient shortcut for loading all the <a href="https://magnusdv.github.io/pedsuite/#core-packages">core packages</a>, making their methods available in the current R session.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://magnusdv.github.io/pedsuite/">pedsuite</a></span><span class="op">)</span>
<span class="co">#&gt; Loading required package: forrel</span>
<span class="co">#&gt; Loading required package: pedtools</span>
<span class="co">#&gt; Loading required package: pedmut</span>
<span class="co">#&gt; Loading required package: pedprobr</span>
<span class="co">#&gt; Loading required package: ribd</span></code></pre></div>
<div id="create-a-pedigree-pedtools" class="section level3">
<h3 class="hasAnchor">
<a href="#create-a-pedigree-pedtools" class="anchor" aria-hidden="true"></a>1. Create a pedigree (<strong>pedtools</strong>)</h3>
<p>We begin by creating and plotting a pedigree with a child of first cousins:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">x</span> <span class="op">=</span> <span class="fu"><a href="https://magnusdv.github.io/pedsuite/reference/ped_basic.html">cousinPed</a></span><span class="op">(</span><span class="fl">1</span>, child <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">x</span><span class="op">)</span></code></pre></div>
<p><img src="pedsuite_files/figure-html/fc-ped1-1.png" width="40%" style="display: block; margin: auto;"></p>
<p>For symmetry let us change the sex of individual 3. We also highlight the child by hatching his symbol, and we only include his ID label in the plot.</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">x</span> <span class="op">=</span> <span class="fu"><a href="https://magnusdv.github.io/pedsuite/reference/getSex.html">swapSex</a></span><span class="op">(</span><span class="va">x</span>, ids <span class="op">=</span> <span class="fl">3</span><span class="op">)</span>
<span class="co">#&gt; Changing sex of spouses as well: 4</span>

<span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">x</span>, labs <span class="op">=</span> <span class="fl">9</span>, hatched <span class="op">=</span> <span class="fl">9</span><span class="op">)</span></code></pre></div>
<p><img src="pedsuite_files/figure-html/fc-ped2-1.png" width="40%" style="display: block; margin: auto;"></p>
</div>
<div id="calculate-the-inbreeding-coefficient-ribd" class="section level3">
<h3 class="hasAnchor">
<a href="#calculate-the-inbreeding-coefficient-ribd" class="anchor" aria-hidden="true"></a>2. Calculate the inbreeding coefficient (<strong>ribd</strong>)</h3>
<p>The inbreeding coefficient <span class="math inline">\(f\)</span> of a pedigree member is defined as the probability of <em>autozygosity</em> at a random autosomal locus. That is, the probability that the two homologous alleles have the same origin within the pedigree.</p>
<p>For a child of first cousins one can work out by pen and paper that <span class="math inline">\(f = 1/16\)</span>. Alternatively, we can calculate it with the <strong>ribd</strong> function <code><a href="https://rdrr.io/pkg/ribd/man/inbreeding.html" class="external-link">inbreeding()</a></code>.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/pkg/ribd/man/inbreeding.html" class="external-link">inbreeding</a></span><span class="op">(</span><span class="va">x</span>, ids <span class="op">=</span> <span class="fl">9</span><span class="op">)</span>
<span class="co">#&gt; [1] 0.0625</span></code></pre></div>
<p>The output agrees with <span class="math inline">\(f = 1/16\)</span>.</p>
</div>
<div id="realised-inbreeding-ibdsim2" class="section level3">
<h3 class="hasAnchor">
<a href="#realised-inbreeding-ibdsim2" class="anchor" aria-hidden="true"></a>3. Realised inbreeding (<strong>ibdsim2</strong>)</h3>
<p>For any particular child of first cousins, the actual autozygous fraction of the genome (except X &amp; Y) is called the coefficient of <em>realised inbreeding</em>, denoted <span class="math inline">\(f_R\)</span>. This may deviate substantially from the pedigree-based expectation <span class="math inline">\(f = 1/16\)</span>.</p>
<p>We can simulate the distribution of <span class="math inline">\(f_R\)</span> with the <strong>ibdsim2</strong> package. Since this is not a core package we must load it separately.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/magnusdv/ibdsim2" class="external-link">ibdsim2</a></span><span class="op">)</span></code></pre></div>
<p>First, we use the function <code><a href="https://magnusdv.github.io/pedsuite/reference/ibdsim.html">ibdsim()</a></code> to simulate the recombination process in the entire pedigree, 200 times:</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">sims</span> <span class="op">=</span> <span class="fu"><a href="https://magnusdv.github.io/pedsuite/reference/ibdsim.html">ibdsim</a></span><span class="op">(</span><span class="va">x</span>, N <span class="op">=</span> <span class="fl">200</span>, seed <span class="op">=</span> <span class="fl">123</span><span class="op">)</span>
<span class="co">#&gt; Simulation parameters:</span>
<span class="co">#&gt; Simulations  : 200</span>
<span class="co">#&gt; Chromosomes  : 1-22</span>
<span class="co">#&gt; Genome length: 2753.93 Mb</span>
<span class="co">#&gt;                2602.29 cM (male)</span>
<span class="co">#&gt;                4180.42 cM (female)</span>
<span class="co">#&gt; Recomb model : chi</span>
<span class="co">#&gt; Target indivs: 1-9</span>
<span class="co">#&gt; Skip recomb  : -</span>
<span class="co">#&gt; Total time used: 9.65 secs</span></code></pre></div>
<p>Now extract the autozygous segments of each simulation.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">fr</span> <span class="op">=</span> <span class="fu"><a href="https://magnusdv.github.io/pedsuite/reference/realised.html">realisedInbreeding</a></span><span class="op">(</span><span class="va">sims</span>, id <span class="op">=</span> <span class="fl">9</span><span class="op">)</span></code></pre></div>
<p>Here is a summary of the first 6 simulations, including the number of segments and various length statistics:</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">fr</span><span class="op">$</span><span class="va">perSimulation</span><span class="op">)</span>
<span class="co">#&gt;   nSeg  meanLen   totLen   maxLen    minLen      fReal</span>
<span class="co">#&gt; 1   18 12.04964 216.8935 35.11428 0.2658601 0.07875777</span>
<span class="co">#&gt; 2   12 14.89156 178.6987 48.86386 0.2469086 0.06488857</span>
<span class="co">#&gt; 3   16 16.54443 264.7109 66.83864 0.4071844 0.09612108</span>
<span class="co">#&gt; 4    9 24.45525 220.0973 61.67048 0.9938968 0.07992111</span>
<span class="co">#&gt; 5   20 15.68533 313.7066 53.35902 1.3336400 0.11391226</span>
<span class="co">#&gt; 6   13 14.04913 182.6387 41.60738 0.1404723 0.06631925</span></code></pre></div>
<p>And here is a histogram of the realised inbreeding coefficients (given in the right-most column above):</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/graphics/hist.html" class="external-link">hist</a></span><span class="op">(</span><span class="va">fr</span><span class="op">$</span><span class="va">perSimulation</span><span class="op">$</span><span class="va">fReal</span>, xlim <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">0</span>, <span class="fl">0.15</span><span class="op">)</span>, breaks <span class="op">=</span> <span class="fl">16</span>, xlab <span class="op">=</span> <span class="st">"f_R"</span>, main <span class="op">=</span> <span class="cn">NULL</span><span class="op">)</span>

<span class="co"># Expected value</span>
<span class="fu"><a href="https://rdrr.io/r/graphics/abline.html" class="external-link">abline</a></span><span class="op">(</span>v <span class="op">=</span> <span class="fl">1</span><span class="op">/</span><span class="fl">16</span>, col <span class="op">=</span> <span class="fl">2</span><span class="op">)</span></code></pre></div>
<p><img src="pedsuite_files/figure-html/fr-hist-1.png" width="80%" style="display: block; margin: auto;"></p>
<p>As we see, the distribution centres around the expectation <span class="math inline">\(f = 1/16 = 0.0625\)</span> (red vertical line) but has substantial spread. The sample standard deviation can be found in <code>fr$stDev</code>, which in our case is 0.024.</p>
</div>
<div id="marker-simulation-forrel" class="section level3">
<h3 class="hasAnchor">
<a href="#marker-simulation-forrel" class="anchor" aria-hidden="true"></a>4. Marker simulation (<strong>forrel</strong>)</h3>
<p>Note that everything we have done so far has been purely theoretical, with no markers involved. In medical and forensic applications we usually work with genetic data in the form of marker genotypes, so let us simulate such a dataset for our family.</p>
<p>The <code><a href="https://rdrr.io/pkg/forrel/man/markerSim.html" class="external-link">markerSim()</a></code> function of the <strong>forrel</strong> package simulates the genotypes of pedigree members for a specific type of markers. For instance, here we produce 500 SNPs with alleles <code>A</code> and <code>B</code> (equally frequent, by default):</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">y</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/forrel/man/markerSim.html" class="external-link">markerSim</a></span><span class="op">(</span><span class="va">x</span>, N <span class="op">=</span> <span class="fl">500</span>, alleles <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"A"</span>, <span class="st">"B"</span><span class="op">)</span><span class="op">)</span>
<span class="co">#&gt; Unconditional simulation of 500 autosomal markers.</span>
<span class="co">#&gt; Individuals: 1, 2, 3, 4, 5, 6, 7, 8, 9</span>
<span class="co">#&gt; Allele frequencies:</span>
<span class="co">#&gt;    A   B</span>
<span class="co">#&gt;  0.5 0.5</span>
<span class="co">#&gt; Mutation model: No </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Simulation finished.</span>
<span class="co">#&gt; Calls to `likelihood()`: 0.</span>
<span class="co">#&gt; Total time used: 0.01 seconds.</span></code></pre></div>
<p>We can see the genotypes of the first few markers by printing <code>y</code> to the console.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">y</span>
<span class="co">#&gt;  id fid mid sex &lt;1&gt; &lt;2&gt; &lt;3&gt; &lt;4&gt; &lt;5&gt;</span>
<span class="co">#&gt;   1   *   *   1 A/B A/B A/B B/B A/A</span>
<span class="co">#&gt;   2   *   *   2 A/B A/B B/B A/B B/B</span>
<span class="co">#&gt;   3   1   2   2 B/B A/B A/B B/B A/B</span>
<span class="co">#&gt;   4   *   *   1 A/A A/A B/B A/A B/B</span>
<span class="co">#&gt;   5   1   2   1 B/B A/B A/B A/B A/B</span>
<span class="co">#&gt;   6   *   *   2 A/B A/B B/B A/B A/B</span>
<span class="co">#&gt;   7   4   3   1 A/B A/A A/B A/B B/B</span>
<span class="co">#&gt;   8   5   6   2 B/B A/B B/B A/B A/B</span>
<span class="co">#&gt;   9   7   8   1 B/B A/A A/B B/B B/B</span>
<span class="co">#&gt; Only 5 (out of 500) markers are shown.</span></code></pre></div>
</div>
<div id="inference-of-pairwise-relationships-forrel" class="section level3">
<h3 class="hasAnchor">
<a href="#inference-of-pairwise-relationships-forrel" class="anchor" aria-hidden="true"></a>5. Inference of pairwise relationships (<strong>forrel</strong>)</h3>
<p>If this was a real dataset, a natural quality control step would be to check correctness of the pedigree. One way to do this is to use the data to estimate each pairwise relationship, and compare the result with the pedigree. The function <code><a href="https://rdrr.io/pkg/forrel/man/checkPairwise.html" class="external-link">checkPairwise()</a></code> does all of this, and presents the result in a relationship triangle.</p>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/pkg/forrel/man/checkPairwise.html" class="external-link">checkPairwise</a></span><span class="op">(</span><span class="va">y</span>, plot <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span>
<span class="co">#&gt; Warning:</span>
<span class="co">#&gt;  Individual '9' is inbred (f = 0.0625)</span>
<span class="co">#&gt; Kappa coefficients are only defined for non-inbred individuals.</span></code></pre></div>
<p><img src="pedsuite_files/figure-html/check-pairwise-1.png" width="60%" style="display: block; margin: auto;"></p>
<p>The plot shows that all pairwise estimates are near their expected location in the triangle. (The yellow symbols correspond to pairs involving the inbred child, which don’t have a well-defined position in the triangle.)</p>
<hr>
</div>
</div>
<div id="where-to-go-from-here" class="section level2">
<h2 class="hasAnchor">
<a href="#where-to-go-from-here" class="anchor" aria-hidden="true"></a>Where to go from here</h2>
<p>If you enjoyed this quick tour and would like more details, you should check out the GitHub README files of the packages that interest you.</p>
<p>There is also the <a href="https://cran.r-project.org/package=pedtools/vignettes/pedtools.html" class="external-link">pedtools vignette</a>, with a lot more details on how to create and plot pedigrees in R.</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Created by Magnus Dehli Vigeland.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link external-link">pkgdown</a> 1.6.1.9001.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
